rust/agents/relayer/src/merkle_tree/builder.rs

```diff
@@ -14,6 +14,7 @@ use crate::prover::{Prover, ProverError};
 /// Struct to sync prover.
 #[derive(Debug)]
 pub struct MerkleTreeBuilder {
+    #[allow(dead_code)]
```

Comment:
?

Reply:
Otherwise the lint-rs workflow was failing